File: 1сем/ОАиП лабы/Common7/IDE/ItemTemplatesCache/CSharp/Web/1049/WebService/WebService.asmx.cs
using System;
using System.Collections.Generic;
$if$ ($targetframeworkversion$ >= 3.5)using System.Linq;
$endif$using System.Web;
using System.Web.Services;

namespace $rootnamespace$
{
    /// <summary>
    /// Сводное описание для $classname$
    /// </summary>
    [WebService(Namespace = "http://tempuri.org/")]
    [WebServiceBinding(ConformsTo = WsiProfiles.BasicProfile1_1)]
    [System.ComponentModel.ToolboxItem(false)]
$if$ ($targetframeworkversion$ >= 3.5)    // Чтобы разрешить вызывать веб-службу из скрипта с помощью ASP.NET AJAX, раскомментируйте следующую строку. 
    // [System.Web.Script.Services.ScriptService]$endif$
    public class $classname$ : System.Web.Services.WebService {

        [WebMethod]
        public string HelloWorld() {
            return "Привет всем!";
        }
    }
}

File: 1сем/ОАиП лабы/Common7/IDE/ItemTemplatesCache/VisualBasic/Web/1049/WebService/WebService.asmx.vb
Imports System.Web.Services
Imports System.Web.Services.Protocols
Imports System.ComponentModel

$if$ ($targetframeworkversion$ >= 3.5)' Чтобы разрешить вызывать веб-службу из скрипта с помощью ASP.NET AJAX, раскомментируйте следующую строку.
' <System.Web.Script.Services.ScriptService()> _
$endif$<System.Web.Services.WebService(Namespace:="http://tempuri.org/")> _
<System.Web.Services.WebServiceBinding(ConformsTo:=WsiProfiles.BasicProfile1_1)> _
<ToolboxItem(False)> _
Public Class $classname$
    Inherits System.Web.Services.WebService

    <WebMethod()> _
    Public Function HelloWorld() As String
       Return "Привет всем!"
    End Function

End Class